lib/todo.rb

```diff
@@ -1,18 +1,33 @@
 require 'cli/ui'
 require 'cli/kit'
 
-CLI::Kit.configure do |config|
-  config.command_registry = CLI::Kit::CommandRegistry.new
-  config.default_command = 'help'
-  config.error_handler = CLI::Kit::ErrorHandler.new
-  config.executor = CLI::Kit::Executor.new
-  config.log_file = "/tmp/todo.log"
-  config.resolver = CLI::Kit::Resolver.new
-  config.tool_name = "todo"
-end
+CLI::UI::StdoutRouter.enable
 
 module Todo
-  autoload :Config,       'todo/config'
-  autoload :Command,      'todo/command'
-  autoload :EntryPoint,   'todo/entry_point'
+  extend CLI::Kit::Autocall
+
+  TOOL_NAME = 'todo'
+  ROOT      = File.expand_path('../..', __FILE__)
```

Comment:
I don't think this is used?

Reply:
No, but I anticipated putting this in the template because it's frequently useful.